resources/stsci.edu/schemas/rotate3d-1.0.0.yaml

```diff
@@ -36,16 +36,19 @@ allOf:
         description: Angle, in degrees.
       direction:
         description: |
-          Sequence of rotation axes: one of `zxz`, `zyz`, `yzy`, `yxy`, `xyx`, `xzx`
+          Sequence of rotation axes: one of 'zxz', 'xyx', 'yzy', 'zyz', 'xzx',
```

Comment:
This is a little silly, but astropy supports direction strings like `xxx`, `xxy`, etc.  Since it's possible to construct models that way, should we support serializing them?  If so, we can express the enum more succinctly with a regular expression validator:

```
direction:
  type: string
  pattern: ^[xyz]{3}$
```

Reply:
I believe we came to the conclusion that only valid Euler angles should be included, not all possible rotation sequences.